ddtrace/llmobs/_telemetry.py

```diff
@@ -10,8 +25,27 @@ def record_llmobs_enabled(error, agentless_enabled, site, start_ns):
         tags.extend([("error", "1"), ("error_type", error)])
     init_time_ms = (time.time_ns() - start_ns) / 1e6
     telemetry_writer.add_distribution_metric(
-        namespace=TELEMETRY_NAMESPACE.MLOBS, name="llmobs.init_time", value=init_time_ms, tags=tuple(tags)
+        namespace=TELEMETRY_NAMESPACE.MLOBS, name=LLMObsTelemetryMetrics.INIT_TIME, value=init_time_ms, tags=tuple(tags)
     )
     telemetry_writer.add_count_metric(
-        namespace=TELEMETRY_NAMESPACE.MLOBS, name="llmobs.enabled", value=1, tags=tuple(tags)
+        namespace=TELEMETRY_NAMESPACE.MLOBS, name=LLMObsTelemetryMetrics.ENABLED, value=1, tags=tuple(tags)
+    )
+
+
+def record_span_event_size(event: LLMObsSpanEvent, event_size: int):
+    span_kind = event.get("meta", {}).get("span.kind", "")
+    integration = _find_integration_from_tags(event.get("tags", []))
+    autoinstrumented = 1 if integration is not None else 0
+    error = 1 if event.get("status") == "error" else 0
+    tags = [
+        ("span_kind", span_kind),
+        ("autoinstrumented", str(autoinstrumented)),
+        ("error", str(error)),
+    ]
+    if integration:
+        tags.append(("integration", integration))
+    if event_size > EVP_EVENT_SIZE_LIMIT:
```

Comment:
instead of reimplementing the truncation logic i'd prefer to emit the metric in the same control flow that we do in the outer function (just to ensure future changes to truncation will be tracked)